### DISCUSSION TITLE
Remove background music and keep fx and voices

### DISCUSSION THREAD

**Discussion by leop2p** [Anjok07/ultimatevocalremovergui#1198] [Q&A]
Hi there!!!

I need a model, which can help me to remove background music and keep only effects and voices. I mean process the audio file, which extracted from a video.

Is there such model in UVR?

Thank you!!!

**Comment by Benji0TheFox** [comment]
So far all I can say is I know one way to keep the voices when removing music, but that's only if

1. The audio is just instrumental along with the talking voice.

2. You don't mind the sound effects being removed along with the instrumental.

Because most of the softwares at the moment can only separate stems into up to like 6 things such as: Vocals, other, bass, piano, drums, guitar

So far no sign of software that can even separate out a single audio track (having the lead and backup in one track) to be separated as individual tracks. 

So, that's all I can tell you.

**Comment by Benji0TheFox** [comment]
Try Stemroller (For vocals, bass, other, and drums) or Ultimate Vocal Remover (the 6 stems way).

**Comment by leop2p** [comment]
> (the 6 stems way).

You mean multi-stem in Ensemble?
